third_party/move/tools/move-decompiler/tests/move-v2-features/enum.exp

```diff
@@ -143,3 +142,6 @@ module 0x99::enum_simple {
         ()
     }
 }
+
```

Comment:
In the corresponding move file for this exp, `0x99::enum_complex::admin_id` does not look like enum match code, which is presumably what this test case is supposed to test?